src/EPR.Calculator.API.Data/DataModels/CalculatorRun.cs

```diff
@@ -19,12 +19,12 @@ public class CalculatorRun
         [Column("name")]
         [StringLength(250)]
         [Required]
-        public string Name { get; set; }
+        public string? Name { get; set; }
```

Comment:
We cannot have Required and Optional?

Reply:
Taking a second look at this, it looks like it can indeed be set to "required", without any other changes to the code needed, other than the unit tests setting a placeholder value.  
I'll change it.

Reply:
Fixed by adding the required modifier